app/products/playbooks/screens/playbook_run/checklist/checklist.tsx

```diff
@@ -184,6 +254,17 @@ const Checklist = ({
                         isDisabled={isFinished || !isParticipant}
                     />
                 ))}
+                {!isFinished && isParticipant && (
+                    <View style={styles.addButton}>
```

Comment:
This is supposed to be a button, but I don't see any touchable. Also, probably you want to use the `Button` component from `components/button`.